monero-harness/tests/wallet.rs

```diff
@@ -61,3 +64,51 @@ async fn fund_transfer_and_check_tx_key() {
 
     assert_that!(res.received).is_equal_to(send_to_bob);
 }
+
+#[tokio::test]
+async fn load_watch_only_wallet() {
+    let tc = Cli::default();
+    let (monero, _containers) = Monero::new(&tc, None, vec!["watch-only".into()])
+        .await
+        .unwrap();
+    let miner_wallet = monero.wallet("miner").unwrap();
+
+    let (_spend_sk, spend_pk) = {
+        let scalar = Scalar::random(&mut OsRng);
+        let sk = PrivateKey::from_scalar(scalar);
+        let pk = PublicKey::from_private_key(&sk);
+
+        (sk, pk)
+    };
+    let (view_sk, view_pk) = {
+        let scalar = Scalar::random(&mut OsRng);
+        let sk = PrivateKey::from_scalar(scalar);
+        let pk = PublicKey::from_private_key(&sk);
+
+        (sk, pk)
+    };
+    let address = Address::standard(Network::Mainnet, spend_pk, view_pk);
+
+    let one_xmr = 1_000_000_000_000;
```

Comment:
If often used in test then it could be a const.

What's the name of the smallest monero unit again?

Reply:
Piconero (or atomic unit) https://web.getmonero.org/resources/moneropedia/atomic-units.html